guardian/content-api-sanity-tests
# Maintenance: remove canary content test

## What does this change?

Removes the canary-injection test because that is handled by prodmon now

## How to test

Deploy and check the logs. Everything should still work fine but there should be no reference to canary content.
The Porter logs, likewise, should show that the canary endpoint is not being hit

## How can we measure success?

Able to deprecate Porter http exposure

## Have we considered potential risks?

n/a.  This test was superceded long ago by Prodmon.
